chart/values.yaml

```diff
@@ -108,10 +110,8 @@ siem_env:
     rollover_alias_zeek: malcolm_network_zeek
     rollover_alias_suricata: malcolm_network_suricata
     search_alias: malcolm_network
-  # Allows user to add a default_pipeline to malcolm_beat_template
```

Comment:
beats template pipeline and lifecycle stuff can now be specified via the variables below